docs/ar/latest/README.md

```diff
@@ -70,75 +70,75 @@
 
 - **متعدد البروتوكولات**
 
-  - [TCP/UDP وكيل](docs/en/latest/stream-proxy.md): ديناميكي TCP/UDP وكيل.
-  - [Dubbo وكيل](docs/en/latest/plugins/dubbo-proxy.md): ديناميكي HTTP to Dubbo وكيل.
-  - [ديناميكي MQTT وكيل](docs/en/latest/plugins/mqtt-proxy.md): يدعم تحميل  MQTT بواسطة  `client_id`, وكلاهما يدعم  MQTT [3.1.\*](http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html), [5.0](https://docs.oasis-open.org/mqtt/mqtt/v5.0/mqtt-v5.0.html).
-  - [gRPC proxy](docs/en/latest/grpc-proxy.md):توكيل حركة مرور  gRPC .
-  - [gRPC transcoding](docs/en/latest/plugins/grpc-transcode.md): يدعم تحويل ترميز البروتوكول بحيث يمكن للعملاء الوصول إلى واجهة برمجة تطبيقات gRPC الخاصة بك باستخدام  HTTP/JSON.
+  - [TCP/UDP وكيل](../../en/latest/stream-proxy.md): ديناميكي TCP/UDP وكيل.
```

Comment:
Hi, I'm not sure if we need to maintain more langs 🤔 Could we keep `English` only?